crates/router/src/routes/app.rs

```diff
@@ -206,15 +206,24 @@ pub struct PaymentMethods;
 #[cfg(feature = "oltp")]
```

Comment:
Remove this feature gate.

Reply:
Good catch. Will make the changes accordingly.